src/types.ts

```diff
@@ -147,6 +147,20 @@ export interface IssuesApiResponse {
   nextPageToken: string;
 }
 
+export interface BigQueryIssue {
+  labels: string[];
```

Comment:
I believe this will come out of BigQuery as a comma-delimited string

Reply:
Done